app/views/teams/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="container">
```

Comment:
We're not adding this container in the other template files. I'd suggest we remove it and, in the future, add the container, row and columns in the main template, so that we render all the pages inside it.